share/rocm/cmake/ROCMInstallTargets.cmake

```diff
@@ -289,6 +289,7 @@ function(rocm_export_targets)
     foreach(INCLUDE ${PARSE_INCLUDE})
         rocm_install(FILES ${INCLUDE} DESTINATION ${CONFIG_PACKAGE_INSTALL_DIR})
         get_filename_component(INCLUDE_BASE ${INCLUDE} NAME)
+        list(APPEND PARSE_INCLUDE_NAME ${INCLUDE_BASE})
```

Comment:
Can you rename the `PARSE_INCLUDE_NAME` variable you're using here, possibly to something like `INCLUDED_FILES`? `PARSE_INCLUDE_NAME` suggests that `INCLUDE_NAME` is an argument to `rocm_export_targets`.